firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java

```diff
@@ -252,9 +252,15 @@ public UpdateTask updateIfNewReleaseAvailable() {
       LogWrapper.getInstance().v("Response in progress");
       return cachedCheckForNewReleaseTask;
     }
+    if (!isTesterSignedIn()) {
+      return Tasks.forException(
+          new FirebaseAppDistributionException(
+              Constants.ErrorMessages.AUTHENTICATION_ERROR, AUTHENTICATION_FAILURE));
+    }
+
     cachedCheckForNewReleaseTask =
-        signInTester()
```

Comment:
Could you remove the comment that starts on line 155? With this PR, we no longer need to qualify why we call signIn before calling checkForNewRelease

Reply:
done